aiida/cmdline/commands/cmd_data/cmd_bands.py

```diff
@@ -44,11 +44,13 @@ def bands():
 @options.WITH_ELEMENTS_EXCLUSIVE()
 @options.FORMULA_MODE()
 def bands_list(elements, elements_exclusive, raw, formula_mode, past_days, groups, all_users):
-    """List BandsData objects."""
-    from aiida.backends.utils import QueryFactory
+    """ List all the bands data types in the database """
```

Comment:
If we change the doc string of this command, we should also change that of the others for consistency. In addition, I think we should start and try to adhere to [PEP 257](https://www.python.org/dev/peps/pep-0257/#id16) for docstring formatting. Do you know why this was excluded from the pre-commit steps?

Reply:
Ah, I didn't realise there was a consistent way that this was done, I'll just revert - I think that's better.  re: PEP 257 you mean with no spaces around the inverted commas?  Don't know how it snuck through the pre-commit.

Reply:
It snuck through because we haven't yet activated/enforced PEP 257 through pre-commit. But yes, it states that single line docstrings should be surrounded in triple double-quotes, without leading or trailing whitespace and a full stop